packages/material-ui/src/Slide/Slide.js

```diff
@@ -302,6 +322,18 @@ Slide.propTypes /* remove-proptypes */ = {
    * @ignore
    */
   style: PropTypes.object,
+  /**
+   * If defined, then element slides in from the edge of the specified element.
+   * @default {
+   *   current: undefined,
+   * }
+   */
+  targetRef: PropTypes.oneOfType([
+    PropTypes.func,
+    PropTypes.shape({
+      current: PropTypes.object,
+    }),
+  ]),
```

Comment:
There is a `refType` utility in `@material-ui/utils`

```suggestion
  targetRef: refType
```

Reply:
This is autogenerated with `yarn proptypes`.

Reply:
Yep, but if it's corrected to `refType`, `yarn proptypes` won't override it.